client/src/components/Form.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+import { Link, useHistory } from "react-router-dom";
+import { createNewPassword } from "../utils/api";
+
+export default function Form() {
+  const history = useHistory();
+  const [passwordName, setPasswordName] = useState("");
+  const [userdata, setUserdata] = useState("");
+  const [passwordValue, setPasswordValue] = useState("");
+
+  function handleNameChange(e) {
+    setPasswordName(e.target.value);
+  }
+
+  function handleUserdataChange(e) {
+    setUserdata(e.target.value);
+  }
+
+  function handlePasswordValueChange(e) {
+    setPasswordValue(e.target.value);
+  }
+
+  async function handleSubmit(passwordName, userdata, passwordValue, e) {
+    e.preventDefault();
```

Comment:
Does this actually work, so you wont get a page refresh? I would implement it in line 32